appveyor.yml

```diff
@@ -3,8 +3,8 @@ shallow_clone: false
 platform: 'x86'
 clone_folder: C:\projects\work
 branches:
-  except:
-    - gh-pages
+  only:
+    - master
```

Comment:
still tests PRs before they merge?

Reply:
Yes. Without the branch limitation, pull requests are tested twice.